src/docs/product/data-management-settings/server-side-sampling/current-limitations.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Current Limitations
+sidebar_order: 20
+description: "Learn about the current limitions of server-side sampling."
+---
+
+Following are the current known limitations and functionality specifics that can create challenges when you're writing server-side sampling rules.
+
+## Supported SDKs & Versions
+
+Server-side sampling is currently only available for the following SDK's:
+
+- Python: 1.8.0 or later
+- JavaScript: 7.5.1 or later
+- Apple Cocoa: 7.3.0 or later
+- Java and Android: 5.2.0 or later
+
+<!-- TO DO version numbers need to be confirmed/updated -->
+
+If your application relies on any Sentry SDK that isn't specified above, then you won’t be able to use server-side sampling.
+
+## Trace rules can only select based on the attributes of the initial transaction
+
+When sampling traces the decision on whether to keep or throw away a trace needs to be taken when the first transaction of the trace is seen. This in turn means that for transaction traces only the attributes of the first transaction can participate in the sampling decision.
+
+In conclusion all trace rules use the attributes (release, environment, and so on) of initiating transaction of the trace.
```

Comment:
Would replace this with a concrete example: If you have a frontend talking to a backend, you can sample the trace by the SDK context available in the frontend, not the backend data.